packages/happy-dom/src/nodes/html-button-element/HTMLButtonElement.ts

```diff
@@ -0,0 +1,82 @@
+import HTMLElement from '../html-element/HTMLElement';
+import IHTMLButtonElement from './IHTMLButtonElement';
+
+/**
+ * HTML Button Element.
+ *
+ * Reference:
+ * https://developer.mozilla.org/en-US/docs/Web/API/HTMLButtonElement.
+ */
+export default class HTMLButtonElement extends HTMLElement implements IHTMLButtonElement {
+	/**
+	 * Returns value.
+	 *
+	 * @returns Value.
+	 */
+	public get value(): string {
+		return this.getAttributeNS(null, 'value');
+	}
+
+	/**
+	 * Sets value.
+	 *
+	 * @param value Value.
+	 */
+	public set value(value: string) {
+		this.setAttributeNS(null, 'value', value);
+	}
+
+	/**
+	 * Returns disabled.
+	 *
+	 * @returns Disabled.
+	 */
+	public get disabled(): boolean {
+		return this.getAttributeNS(null, 'disabled') !== null;
+	}
+
+	/**
+	 * Sets disabled.
+	 *
+	 * @param disabled Disabled.
+	 */
+	public set disabled(disabled: boolean) {
+		if (!disabled) {
+			this.removeAttributeNS(null, 'disabled');
+		} else {
+			this.setAttributeNS(null, 'disabled', '');
+		}
+	}
+
+	/**
+	 * Returns type
+	 *
+	 * @returns Type
+	 */
+	public get type(): string {
+		return this.sanitizeType(this.getAttributeNS(null, 'type'));
+	}
+
+	/**
+	 * Sets type
+	 *
+	 * @param v Type
+	 */
+	public set type(v: string) {
+		this.setAttributeNS(null, 'type', this.sanitizeType(v));
+	}
+
+	/**
+	 *
+	 * @param type
+	 */
+	private sanitizeType(type: string): string {
```

Comment:
```suggestion
	protected _sanitizeType(type: string): string {
```

Reply:
As the methods will be available on the instances of the button elements, I think it is good if we make it clear that they are not part of the browser standard by adding an underline as prefix.

We could also create a utility or move this logic to `public get type()` as well.